backend/src/routes/updates-route.ts

```diff
@@ -20,5 +22,33 @@ router.get("/", [], async (req: Request, res: Response) => {
 
 });
 
+////TESTING ONLY
```

Comment:
testing only?

Reply:
yes cuz in actual app it will go in a scheduler to make it time to time